docs/quick-start.md

```diff
@@ -71,9 +71,10 @@ spec:
     kind: "Deployment"
     name: "gke-cloud-sql-quickstart"
   instances:
-  - connectionString: "<INSTANCE_CONNECTION_NAME>"
-    portEnvName: "DB_PORT"
-    hostEnvName: "INSTANCE_HOST"
+    - connectionString: "<INSTANCE_CONNECTION_NAME>"
+      portEnvName: "DB_PORT"
+      hostEnvName: "INSTANCE_HOST"
+      autoIAMAuthN: True
```

Comment:
Would you update the PR without the `autoIAMAuthN: True` line?

Reply:
Done